modules/silauth/src/Auth/Source/time/UtcTime.php

```diff
@@ -4,14 +4,15 @@
 
 use DateTime;
 use DateTimeZone;
+use Exception;
 use InvalidArgumentException;
 
 class UtcTime
 {
     const DATE_TIME_FORMAT = 'Y-m-d H:i:s';
 
-    private $utc;
-    private $dateTime;
+    private DateTimeZone $utc;
+    private DateTime $dateTime;
```

Comment:
Is this always initialized before used?

Reply:
Yes, this also is set in the constructor.